comp/core/status/component.go

```diff
@@ -25,6 +25,12 @@ type Component interface {
 	GetStatusBySection(section string, format string, verbose bool) ([]byte, error)
 }
 
+// Params store configurable options for the status component
+type Params struct {
+	Flavor        string
```

Comment:
Flavor being such a small package, would it not be easier to convert it as a module to rather that take is as a parameter ?

Reply:
Done https://github.com/DataDog/datadog-agent/pull/22153/commits/02f45bc9441521dc1f01b474e53e6fbd63748588 and https://github.com/DataDog/datadog-agent/pull/22153/commits/57790ac17fe42f6e75f0f36041322459028a169a